plugins/org.eclipse.n4js.n4mf.model/model/N4MF.xcore

```diff
@@ -298,22 +176,25 @@ class BootstrapModule {
  * implementation provided by the runtime.
  */
 enum ModuleFilterType {
-	NO_VALIDATE
-	, NO_MODULE_WRAPPING
-}
-
-/**
- * Class for representing the extended runtime environment project of the current project.
- */
-class ExtendedRuntimeEnvironment {
-	contains ProjectReference extendedRuntimeEnvironment
+	NO_VALIDATE,
+	NO_MODULE_WRAPPING
 }
 
 class ProjectReference {
-	contains SimpleProjectDescription project
-}
 
-class SimpleProjectDependency extends ProjectReference {
+	/**
+	 * the project ID
+	 */
+	String projectId
+
+	// TODO is following property and operation really required?
```

Comment:
Yeah, probably not because the `projectID` should uniquely identify the project, shouldn't it?

Reply:
different vendors can provide projects with the same ID. That is also what npm allows.

Reply:
yeah, this is one of the things that need to be discussed when aligining manifest to package.json